### DISCUSSION TITLE
[DISCUSS] Releasing 5.1.0?

### DISCUSSION THREAD

**Discussion by tisonkun** [wyyerd/pulsar-rs#257] [General]
We have several great patches applied from the last release and I'd like to make a 5.1.0 release now.

cc @FlorentinDUBOIS @klaatu01 What do you think? Are there other concerns or pending issues expected to be resolved?

**Comment by klaatu01** [comment]
I am confident in the AuthChanges, for what it's worth my team has been using my fork in prod for over a week without issue. Would be great to be tracking the official releases again!

**Reply by tisonkun** [reply]
Thanks for your feedback! If there's no more objection today, I'll make a release later and notify here.

**Reply by tisonkun** [reply]
Released as:

* https://github.com/streamnative/pulsar-rs/releases/tag/v5.1.0
* https://crates.io/crates/pulsar/5.1.0

**Comment by FlorentinDUBOIS** [comment]
Yes, it could be great to release a new version :)
